build.sbt

```
@@ -9,3 +9,17 @@ scalacOptions := Seq(
 val gatlingVersion = "3.7.3"
 libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion % "test,it"
 libraryDependencies += "io.gatling"            % "gatling-test-framework"    % gatlingVersion % "test,it"
+
+// Enterprise Cloud configurations (https://cloud.gatling.io/)
+// Reference: https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#working-with-gatling-enterprise-cloud
+
+// Alternative (preferred for security reasons):
```

I don't get it. What's this? Why doesn't it go in the aforementioned page?

---

As @guilgaly state:
> and maybe also keep the recommendation to use the env var for the api token (to make sure we're not encouraging people to commit the token to their source repo)

---

0 doc here. Everything on the website.

---

If so, maybe at least:
```suggestion
// For security purposes, check other API token configuration methods: 
// https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#api-tokens
```